src/libraries/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -96,7 +96,6 @@ public abstract partial class AsymmetricSignatureFormatter
         public abstract void SetHashAlgorithm(string strName);
         public abstract void SetKey(System.Security.Cryptography.AsymmetricAlgorithm key);
     }
-    [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
```

Comment:
Getting warnings `'CryptoConfig.CreateFromName(string)' is unsupported on 'browser'` for Create(string hashName) overload of SHA1.cs, SHA256.cs, SHA384.cs, SHA512.cs. This seems to be shared utility class between crypto algorithms and doesn't seem platform-dependent type (removing the attribute doesn't cause any warning) 
```
D:\dotnet\runtime\src\libraries\System.Security.Cryptography.Algorithms\src\System\Security\Cryptography\SHA512.cs(27,71): error CA1416: 'CryptoConfig.CreateFromName(string)' is unsupported on 'browser' [D:\dotnet\runtime\src\libraries\System.Security.Cryptography.Algorithms\src\System.Security.Cryptography.Algorithms.csproj]
D:\dotnet\runtime\src\libraries\System.Security.Cryptography.Algorithms\src\System\Security\Cryptography\SHA384.cs(27,71): error CA1416: 'CryptoConfig.CreateFromName(string)' is unsupported on 'browser' [D:\dotnet\runtime\src\libraries\System.Security.Cryptography.Algorithms\src\System.Security.Cryptography.Algorithms.csproj]
D:\dotnet\runtime\src\libraries\System.Security.Cryptography.Algorithms\src\System\Security\Cryptography\SHA256.cs(27,71): error CA1416: 'CryptoConfig.CreateFromName(string)' is unsupported on 'browser' [D:\dotnet\runtime\src\libraries\System.Security.Cryptography.Algorithms\src\System.Security.Cryptography.Algorithms.csproj]
D:\dotnet\runtime\src\libraries\System.Security.Cryptography.Algorithms\src\System\Security\Cryptography\SHA1.cs(28,67): error CA1416: 'CryptoConfig.CreateFromName(string)' is unsupported on 'browser' [D:\dotnet\runtime\src\libraries\System.Security.Cryptography.Algorithms\src\System.Security.Cryptography.Algorithms.csproj]
```

Reply:
We might want to port it to 5.0 cc @jeffhandley

Reply:
We just took some more changes like this into 5.0, but I don't know that we'd be able to get more changes through.  @lewing @MaximLipnin -- what are your thoughts on this one?  We marked the entire `CryptoConfig` class as unsupported on browser, but `SHA512`, `SHA384`, `SHA256`, and `SHA1` all use it.

Reply:
BTW, my recommendation would be to take this change in 6.0 but not port it to 5.0.

Reply:
@jeffhandley @buyaa-n https://github.com/dotnet/runtime/pull/43611 It looks safe to me, I'm happy to backport it now for further discussion.